src/lib/operators/multi_predicate_join/multi_predicate_join_evaluator.cpp

```diff
@@ -0,0 +1,77 @@
+#include "multi_predicate_join_evaluator.hpp"
```

Comment:
Do these two files warrant their own folder?

Reply:
I wanted only actual operator classes in `lib/operator/` with utility classes being in subfolders. I think this is fine, better than polluting `lib/operator/` with non-operators.

Reply:
Ok.